AndroidRunner/Plugins/batterymanager/README.md

```diff
@@ -0,0 +1,58 @@
+# BatteryManager Plugin
+This plugin uses the Android BatteryManager API to gather battery related data. The plugin works using the 
+[BatteryManager companion app](https://github.com/Raiduy/ar-batterymanager-script/releases). 
```

Comment:
Fork this repo into the S2 group organization and work in the forked repo from now on. 
This is to have everything in one single organization, without fragmenting our dependencies too much.